src/main/java/nl/dtl/fairmetadata4j/utils/MetadataUtils.java

```diff
@@ -342,8 +342,6 @@
         LOGGER.info("Adding FDP metadata properties to the rdf model");
         ValueFactory f = SimpleValueFactory.getInstance();
         addStatement(model, metadata.getUri(), RDF.TYPE, R3D.REPOSITORY);
-        IRI swaggerURL = f.createIRI(metadata.getUri().toString() + "/swagger-ui.html");
-        metadata.setSwaggerDoc(swaggerURL);
```

Comment:
I assume there were no tests that covered this? 👼

Reply:
Yes, there are no test to check which predicate is used.